card/introduction.mdx

```diff
@@ -1,87 +1,119 @@
 ---
-title: 'Introduction'
-description: 'Text, title, and styling in standard markdown'
+title: 'Overview'
+description: 'The SwyftPay Cards API provides a powerful, secure, and easy-to-use set of endpoints for card management, including creation, retrieval, freezing, and termination of cards. This guide will cover essential operations to help you manage card transactions within your applications.'
 ---
 
-## Titles
+### Authentication
 
-Best used for section headers.
+All interactions with the Cards API require authenticated requests. Include an authorization token in your request headers. Learn how to obtain your [Bearer `<TOKEN>`](#authentication) in our authentication guide.
```

Comment:
To avoid repetition, let's move this to Authentication under get started, something like this https://maplerad.dev/docs/authentication